demo/00Index

```diff
@@ -1 +1,2 @@
 example_1		Basic package usage
+example_2       Using data.frame as input data
```

Comment:
Lovely that you added a 2nd example!   But I can't run it:

```sh
edd@rob:~/git/rvwgsoc/demo(feature/data-parser)$ Rscript example_2.R 
Converting data.frame to VW frame
Error in vwtrain(vwmodel, data = df_train, namespaces = list(NS1 = list("sex",  : 
  Evaluation error: invalid 'times' argument.
Execution halted
edd@rob:~/git/rvwgsoc/demo(feature/data-parser)$ 
```

(Obviously after installing this version first.)

Reply:
I see, I will take a closer look